ls1intum/Artemis
# Text Assessment V2: Complaints

### Checklist

### Description
In this PR, we handle complaints in the new Text Assessment Interface.

### Steps for Testing

1. **[Student]** Submit Text Exercise
2. **[Tutor 1]** Navigate to Tutor Dashboard and Review Assessment
    (Update URL manually from `.../submissions/...` to `.../submissions-new/...`)
3. **[Student]** Submit Complaint
4. **[Tutor 2]** Find Complaint in Tutor Dashboard and open it
    (Update URL manually from `.../submissions/...` to `.../submissions-new/...`)
5. **[Tutor 2]** Update Assessment and respond to Complaint (end of page)
6. **[Student]** See updated result.

### Screenshots
<img width="745" alt="image" src="https://user-images.githubusercontent.com/6382716/79451549-00740d80-7fe7-11ea-957a-26075dc55483.png">
